packages/server/utils/OpenAIServerManager.ts

```diff
@@ -298,21 +298,55 @@ class OpenAIServerManager {
     if (!this.openAIApi) return null
     const meetingURL = 'https://action.parabol.co/meet/'
     const promptForMeetingData = `
-    You work at a start-up and you need to discover behavioral trends for a given team.
-    Below is a list of reflection topics in YAML format from meetings over recent months.
-    You should describe the situation in two sections with no more than 3 bullet points each.
-    The first section should describe the team's positive behavior in bullet points. One bullet point should cite a direct quote from the meeting, attributing it to the person who wrote it.
-    The second section should pick out one or two examples of the team's negative behavior and you should cite a direct quote from the meeting, attributing it to the person who wrote it.
-    When citing the quote, include the meetingId in the format of https://action.parabol.co/meet/[meetingId].
-    Prioritize topics with more votes.
-    Be sure that each author is only mentioned once.
-    Your tone should be kind and straight forward. Use plain English. No yapping.
-    Return the output as a JSON object with the following structure:
-    {
-      "wins": ["bullet point 1", "bullet point 2", "bullet point 3"],
-      "challenges": ["bullet point 1", "bullet point 2"]
-    }
-    `
+You are a Team Lead and want to use your meeting data to help write a report on your team's performance. You care about team productivity, morale, roadblocks, relationships, and progress against goals. Below is a list of retrospective meeting summaries (in YAML format) from the past several months.
+
+**Task:**
+Analyze the provided meeting data and identify patterns in teamwork and collaboration. Focus on "wins" and "challenges" that appear in at two or more different meetings, prioritizing trends that appear in the highest number of meetings. Reference those meetings by hyperlink. Prioritize trends that have received the most combined votes, if that information is available.
+
+**Output Format:**
+Return the analysis as a JSON object with this structure:
+{
+  "wins": ["bullet point 1", "bullet point 2", "bullet point 3"],
+  "challenges": ["bullet point 1", "bullet point 2", "bullet point 3"]
+}
+
+**Instructions:**
+1. **Wins (3 bullet points)**:
+   - Highlight positive trends or patterns observed across multiple meetings.
+   - Include at least one direct quote from one meeting, attributing it to its author.
+   - Link to the referenced meeting(s) using the format:
+     [<meeting title>](https://action.parabol.co/meet/[meetingId])
```

Comment:
-1 Let's use the `meetingURL` variable from above here, then it's easier to plug in the real URL from the environment